pkg/minikube/constants/constants.go

```diff
@@ -37,6 +37,7 @@ const (
 	UpdateMarkerFileName             = "updated"
 	DefaultMachineName               = "minishift"
 	DefaultProfileName               = "minishift"
+	DefaultTimeZone                  = "America/New_York" // This is what we have in our ISO kickstart template.
```

Comment:
I think it makes more sense to amke this UTC / Zulu time (or also called the Coordinated Universal Time)‎. this is plain GMT which does not care about DST, etc.

Reply:
But it also makes more sense to enforce this from the ISO

Reply:
not UTC ?

Reply:
@gbraad since it is not enforced to ISO so I didn't put it here, which will make it confusing like user can see default timezone here as `UTC` but when see in the ISO then it see as `America/New_York` I think first we should enforce the ISO and then update it here.